server/app/playbook.go

```diff
@@ -235,17 +235,17 @@ type PlaybookService interface {
 	// Restores a deleted playbook
 	Restore(playbook Playbook, userID string) error
 
-	// Follow method lets user follow a all runs of a specific playbook
-	Follow(playbookID, userID string) error
+	// AutoFollow method lets user follow a all runs of a specific playbook
```

Comment:
Should we push through to terminology too, like:
```suggestion
	// AutoFollow method lets a user automatically follow a all runs of a specific playbook
```